packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -809,8 +809,9 @@ mixin WidgetInspectorService {
   void _registerSignalServiceExtension({
     required String name,
     required FutureOr<Object?> Function() callback,
+    required RegisterServiceExtensionCallback registration,
   }) {
-    registerServiceExtension(
+    registration(
```

Comment:
these all need to remain calls to 
registerServiceExtension as that is prepending the name with inspector.` 
Concerning if tests are still passing with this change. I would hope at least some tests are verifying that the correct service extension names are used.